q2_feature_table/_summarize/summarize_assets/sample-frequency-detail.html

```diff
@@ -262,6 +269,11 @@ <h3>Plot Controls</h3>
     }
     updateSliderVal(val);
   }
+
+  // Parse a string that may contain commas into a base 10 integer
+  function strToInt(val) {
+    return parseInt(val.replaceAll(",", ""), 10);
```

Comment:
The only risk here is the formatting could depend on LC_NUMERIC which will define the customary radix operator for the locale. Since this happens in Python-land there's not a convenient way to notice.

Reply:
Can't we just also replace "."?